storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -1178,7 +1212,7 @@
           (FileUtils/copyDirectory (File. (.getFile url)) (File. target-dir)))))))
 
 (defmethod launch-worker
-    :local [supervisor storm-id port worker-id mem-onheap]
+    :local [supervisor storm-id port worker-id resources]
```

Comment:
I think when I made my comment I thought we had simply added an unused parameter, but that was not the case. Yes, we should keep things consistent as you have done, so disregard.